### PR TITLE
Fb ignore missing objects

### PR DESCRIPTION
**Pull request by alonbrody**
When an error code 100 arrives it means that a specific object is missing. We can skip it in case that the user wanted to skip the missing/deleted objects.

This change also has @oshribin changes.

@alonweissfeld @liorrozen

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -8,6 +8,26 @@ var METRICS = require( "./metric-list" );
...
+    }
+    var _err = { message: 'object foo does not exist' };
+    var response = { "myApp": { error: _err, name: "myApp", data: dataGenerator( 1, null ) } }
+
+    before( initialize( result, response, source ) )
+    after( reset )
+
+    it( 'Skip missing data', function () {
+        var dataSize = Object.keys( result.data[ 0 ] ).length;
+        // the data size should be as the size of metrics + 3 columns ( date, name, id )
```
Comment (alonweissfeld) [inline test.js]: @alonbrody Isn't it ` .. + 2 columns`?

**Comment by alonweissfeld** [comment]
@alonbrody Looks good (I only reviewed your commits), but let's wait first for #7 merge first.

**Comment by alonbrody** [comment]
@alonweissfeld I've adjusted the comment to the current data structure

**Comment by alonbrody** [comment]
@alonweissfeld @liorrozen Closing and reopening to see the actual changes in this PR

**Pull request closed without merging**